src/Emanote/Model/Stork/Index.hs

```diff
@@ -98,7 +110,29 @@ fileCodec =
     <*> Toml.text "url" .= fileUrl
     <*> Toml.text "title" .= fileTitle
 
+showHandling :: Handling -> Text
+showHandling handling = case handling of
+  Handling_Ignore -> "Ignore"
+  Handling_Omit -> "Omit"
+  Handling_Parse -> "Parse"
+
+parseHandling :: Text -> Either Text Handling
+parseHandling handling = case handling of
+  "Ignore" -> Right Handling_Ignore
+  "Omit" -> Right Handling_Omit
+  "Parse" -> Right Handling_Parse
+  other -> Left $ "Unsupport value for frontmatter handling: " <> other
+
+handlingCodec :: Toml.Key -> TomlCodec Handling
+handlingCodec = textBy showHandling parseHandling
```

Comment:
It seems Tomland needs you to specify how to parse even if you are only generating. If we don't want to write code for that, we could use

```haskell
handlingCodec = textBy showHandling (\_ -> error "Not implemented")
```
or
```haskell
handlingCodec = textBy showHandling(\_ -> Left "Not implemented")
```

But I'm not sure if that's how you usually handle these cases.

Reply:
Partial functions should be avoided as much as possible. More code is not necessarily bad. Here, Tomland is a *bidirectional* serializer, so it makes sense.